<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProjectDialog</class>
 <widget class="QDialog" name="AddProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter un projet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/FactDev</normaloff>:/icons/FactDev</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="lbDescription">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbProjectName">
     <property name="text">
      <string>Projet</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="leDescription">
     <property name="toolTip">
      <string>Description du contenu du projet</string>
     </property>
     <property name="placeholderText">
      <string>Une courte description du projet</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Gui::Widgets::RateWidget" name="wdgRate" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" rowspan="2" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnValid">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/img/ok_valid.png</normaloff>:/icons/img/ok_valid.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/icons/img/cancel.png</normaloff>:/icons/img/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="Gui::Widgets::searchWidget" name="wdgSearch" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CheckUntilField" name="leNameProject">
     <property name="toolTip">
      <string>Nom du projet</string>
     </property>
     <property name="placeholderText">
      <string>Nom du projet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::Widgets::CheckUntilField</class>
   <extends>QLineEdit</extends>
   <header>gui/widgets/checkfields/checkuntilfield.h</header>
   <slots>
    <slot>fieldTextChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::RateWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/ratewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Gui::Widgets::searchWidget</class>
   <extends>QWidget</extends>
   <header>gui/widgets/searchwidget.h</header>
   <container>1</container>
   <slots>
    <signal>selectCustomer()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leNameProject</tabstop>
  <tabstop>leDescription</tabstop>
  <tabstop>btnValid</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnValid</sender>
   <signal>pressed()</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>released()</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>516</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNameProject</sender>
   <signal>textChanged(QString)</signal>
   <receiver>leNameProject</receiver>
   <slot>fieldTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leNameProject</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wdgSearch</sender>
   <signal>selectCustomer()</signal>
   <receiver>AddProjectDialog</receiver>
   <slot>checkFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkFields()</slot>
 </slots>
</ui>
